lib/mutant/mutator/node/literal/range.rb

```diff
@@ -11,7 +11,7 @@ class Range < self
             erange: :irange
           }.freeze
 
-          children :start, :_end
+          children :lower_bound, :upper_bound
```

Comment:
I think you can remove these named children and in `#dispatch` simply go for `#emit_children_mutations`.